src/DDTrace/Integrations/Integration.php

```diff
@@ -8,15 +8,25 @@
 
 abstract class Integration
 {
+    // Possible statuses for the concrete:
+    //   - NOT_LOADED   : It has not been loaded, but it may be loaded at a future time if the preconditions match
+    //   - LOADED       : It has been loaded, no more work required.
+    //   - NOT_AVAILABLE: Prerequisites are not matched and won't be matched in the future.
+    const NOT_LOADED = 0;
+    const LOADED = 1;
+    const NOT_AVAILABLE = 2;
+
     const CLASS_NAME = '';
 
     public static function load()
     {
         if (!class_exists(static::CLASS_NAME)) {
-            return;
+            return Integration::NOT_LOADED;
```

Comment:
Just a little tweak :)

```suggestion
            return self::NOT_LOADED;
```